src/linker/Linker.Dataflow/JsonFlowAnnotationSource.cs

```diff
@@ -105,7 +113,9 @@ private void Initialize(LinkContext context, string jsonFile)
 						foreach (var member in typeElement.Value.EnumerateObject ()) {
 							string memberName = member.Name;
 
-							if (member.Value.ValueKind == JsonValueKind.Object) {
+							// Let's assume fields cannot have '(' in their names and cross our fingers nobody names
```

Comment:
We can lookup the name on the type - if it's a field/event/property we would know right away. I don't have a problem with using parenthesis here, just that the comment feels weird.